scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Let users know this is an actual example.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>example  was replaced with comments</p>